src/patchelf.cc

```diff
@@ -2408,6 +2642,8 @@ static void showHelp(const std::string & progName)
   [--rename-dynamic-symbols NAME_MAP_FILE]\tRenames dynamic symbols. The map file should contain two symbols (old_name new_name) per line\n\
   [--output FILE]\n\
   [--debug]\n\
+  [--try-layout-engine]\n\
+  [--only-layout-engine]\n\
```

Comment:
TODO: help text.

Reply:
Also this will eventually need some manpage text.

Reply:
I wonder what is the best approach.
If this is to be merged, perhaps we can avoid documenting it in a first moment until more testing is done.
Then at some point we enable and document it. And further down the line we remove that fallback.

But before any of this happens, I'm currently working on testing.

I'm running some stress test (running Patchelf on all binaries in the machine) and I found a couple of issues both in the current implementation and the layout engine.

Moreover I will go through the existing code making sure that at least one test fails of we disable any of the special ifs in the code.

Reply:
It's easier to test and receive if there is documentation. If the interface is not stable than a better approach is to declare this as an unstable interface that may change.